src/app/logic/correctAnswer.js

```diff
@@ -0,0 +1,7 @@
+const isAnswerCorrect = (correctAnswer, chosenAnswer) => {
```

Comment:
Perhaps you could name the file exactly the same as the name of the function?

Reply:
Yeah, you are right.